### PR TITLE
[SofaMiscFem] Fix FastTetrahedralCorotationalFF topology change

### PR DESCRIPTION
**Pull request by epernod**
Some changes to be able to handle tetra removal.

______________________________________________________

This PR: 
- [x] builds with SUCCESS for all platforms on the CI.
- [x] does not generate new warnings.
- [x] does not generate new unit test failures.
- [x] does not generate new scene test failures.
- [x] does not break API compatibility.
- [x] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by epernod** [comment]
[ci-build][with-scene-tests]

**Comment by guparan** [comment]
[ci-build][with-scene-tests]

**Comment by damienmarchal** [comment]
Currently this PR makes forcefield/fastTetrahedralCorotationalFEM crash one the CI.

File: modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl [hunk 1]
```diff
@@ -155,6 +155,11 @@ template <class DataTypes> FastTetrahedralCorotationalForceField<DataTypes>::Fas
     , f_youngModulus(initData(&f_youngModulus,(Real)1000.,"youngModulus","Young modulus in Hooke's law"))
     , lambda(0)
     , mu(0)
+    , f_drawing(initData(&f_drawing, true, "drawing", " draw the forcefield if true"))
```
Comment (damienmarchal) [inline modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl:158]: This rise some question to me about the general consistency of drawing in Sofa...but this is far 
outside the scope of this PR.

File: modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl [hunk 2]
```diff
@@ -155,6 +155,11 @@ template <class DataTypes> FastTetrahedralCorotationalForceField<DataTypes>::Fas
...
+    , drawColor1(initData(&drawColor1, defaulttype::Vec4f(0.0f, 0.0f, 1.0f, 1.0f), "drawColor1", " draw color for faces 1"))
```
Comment (damienmarchal) [inline modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl:159]: Why not a vector of color instead of 4 data ?

**Comment by epernod** [comment]
Ok I'll check that crash for next dev meeting.

File: modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl (hunk 1 above)
Comment (epernod) [reply modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl:158]: I took the code that is in TetrahedralCorotationalForcefield to be consistent. Is it a good way to do it, this is another discussion...

File: modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl (hunk 2 above)
Comment (epernod) [reply modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl:159]: same answer as above.

**Comment by guparan** [comment]
The scene is not crashing, it timeouts. Obviously lowering the number of CG iterations avoids the timeout but does not solve the problem...
I checked the scene before this PR and it is as slow as after. So why wasn't it timeout-ing before?

**Comment by guparan** [comment]
Ok found it: the scene-tests are not performing in default mode (100 iterations with a timeout of 60s for a Debug build) because FastTetrahedronCorotationalForceField.scn is listed with custom values in examples/.scene-tests config file.
Renaming the scene according to b061a51606f9dbe6a8bb9305eb6a95af0aad6bf5 should calm down the CI ;-)

**Comment by guparan** [comment]
[ci-build][with-scene-tests]

**Comment by guparan** [comment]
[ci-build][with-scene-tests] (please CI don't die on me)

**Pull request merged**